azure/template.json

```diff
@@ -364,7 +376,13 @@
                     },
                     "storageAccountName": {
                         "value": "[variables('storageAccountName')]"
-                    }
+                    },
+		    "storageAutoGrow": {
+			"value": "[parameters('databaseStorageAutoGrow')]"
```

Comment:
Indentation?

Reply:
Not sure what happened to that indentation there, I'll get that fixed now.
Yes, the removal of that task does fix the deployment stages of the pipeline. We clearly stopped using those outputs, or possibly never actually used them, and the redundant task was not removed.

Reply:
All fixed :)